routes/section.segment.router.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express');
+const sectionSegmentService = require('../services/section.segment.service');
+// const Segment = require('../models/segment.schema');
+
+const router = express.Router();
+
+// TODO: Create services in a new file
+// Example
+// testing
+/*
+router.post('/', async (req, res) => {
+  try {
+    const test = new Segment({ _id: 'testing', name: 'testing' });
+    const mongoResponse = await test.save();
+    // const mongoResponse = await sectionSegmentService.createSection(req.body);
+    res.status(200).send(mongoResponse);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send(new Error('Operation failed'));
+  }
+});
+*/
+
+// Create section
+router.post('/section', async (req, res) => {
+  try {
+    const mongoResponse = await sectionSegmentService.createSection(req.body);
+    res.status(200).send(mongoResponse);
+  } catch (err) {
+    // console.error(err);
+    res.status(500).json({ message: err.message });
+  }
+});
+
+// Create segments for section
+router.post('/segment', async (req, res) => {
+  try {
+    // const mongoResponse = await sectionSegmentService.createSegment(req.body);
+    // res.status(200).send(mongoResponse);
+  } catch (err) {
+    // console.error(err);
+    res.status(500).json({ message: err.message });
+  }
+});
+
+// Edit section
+router.put('/section/:id', async (req, res) => {
+  const { id } = req.params;
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'id' is assigned a value but never used.